index.js

```diff
@@ -129,7 +137,7 @@ exports.parse = function (str, opts) {
 		return ret;
 	}
 
-	str = str.trim().replace(/^(\?|#|&)/, '');
+	str = str.trim().replace(/^(\?|#|&|%)/, '');
```

Comment:
This change needs to be removed. This replace is meant to remove `?` from the start of a query string.